code/game/objects/buckling.dm

```diff
@@ -28,16 +35,25 @@
 //Cleanup
 /atom/movable/Destroy()
 	. = ..()
-	unbuckle_mob()
+	unbuckle_all_mobs(force=TRUE)
 
 /atom/movable/proc/has_buckled_mobs()
-	if(buckled_mob)
+	if(buckled_mobs && buckled_mobs.len)
 		return TRUE
 	return FALSE
 
 //procs that handle the actual buckling and unbuckling
-/atom/movable/proc/buckle_mob(mob/living/M, force = 0)
-	if((!can_buckle && !force)|| !istype(M) || (M.loc != loc) || M.buckled || M.buckled_mob || buckled_mob || (buckle_requires_restraints && !M.restrained()) || M == src)
+/atom/movable/proc/buckle_mob(mob/living/M, force = FALSE, check_loc = TRUE)
```

Comment:
Does this allow the buckling of simple_animals and cyborgs to things? If so, can they unbuckle themselves? If yes and no, respectively, is that balanced? I'm thinking of for example, buckling a secborg to a chair in the bar to effectively neutralize it until a human can come release it.

Reply:
i will check. probaly not to simple animals.

Reply:
borgs can unbuckle themselves.

Reply:
Have you tested this? Can simple animals unbuckle themselves?

Reply:
Doesn't resist normally unbuckle?

Reply:
yes and yes